src/main/java/org/commcare/formplayer/session/FormSession.java

```diff
@@ -89,14 +89,16 @@ public class FormSession {
     private boolean shouldAutoSubmit;
     private boolean suppressAutosync;
     private boolean shouldSkipFullFormValidation;
+    private String windowWidth;
 
     public FormSession(SerializableFormSession session,
             RestoreFactory restoreFactory,
             FormSendCalloutHandler formSendCalloutHandler,
             FormplayerStorageFactory storageFactory,
             @Nullable CommCareSession commCareSession,
             RemoteInstanceFetcher instanceFetcher,
-            FormDefinitionService formDefinitionService) throws Exception {
+            FormDefinitionService formDefinitionService,
+            String windowWidth) throws Exception {
```

Comment:
Instead of passing `windowWidth` directly can we pass a Object say `MetaSessionContext` that can host all different session variables, otherwise adding each parameter to session meta data will cause addition of new variable in method signatures for all new params.

Reply:
I'm getting all of the constructors accepting `windowWidth` from beans before I shift every thing to this new format.